csrankings-b.csv

```diff
@@ -821,6 +821,7 @@ Brian Y. Chen,Lehigh University,http://www.cse.lehigh.edu/~chen,CrqkG-4AAAAJ
 Brian Y. Lim,National University of Singapore,http://www.brianlim.net,_bza0AoAAAAJ
 Briane Paul Samson,De La Salle University,https://brianesamson.com,oUJrZ_sAAAAJ
 Briane Paul V. Samson,De La Salle University,https://brianesamson.com,oUJrZ_sAAAAJ
+Brice Ménard,Johns Hopkins University,https://physics-astronomy.jhu.edu/directory/brice-menard/,vZdmdM8AAAAJ

```

Comment:
<img width="1306" alt="Screen Shot 2024-08-05 at 9 59 22 AM" src="https://github.com/user-attachments/assets/4d99a3ae-6eb7-408f-b8b3-b06436dba2b5">

On this page: https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/